sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2814,6 +2816,20 @@ object Radar {
             })
     }
 
+    internal fun logOpenedAppConversion() {
+        // if opened_app has been logged in the last 250 milliseconds, don't log it again
+        val timestamp = System.currentTimeMillis()
+        val lastAppOpenTime = RadarSettings.getLastAppOpenTimeMillis(context)
+        if (timestamp - lastAppOpenTime > 250) {
```

Comment:
this cooldown was added to prevent the following duplication bug:
```
User opens app
-> initialize()
    -> updateSessionId()
        -> log opened_app
-> onActivityResumed()
    -> val updated = updateSessionId(), returns false and doesn't log opened_app because sessionId was just updated
    -> if (!updated) log opened_app, opened_app is logged a second time for one app open
```

Reply:
Hmmm, why is 250 milliseconds the right number here?

My default guess would have been 1000 ms / 1s

(250ms could very well be fine, just trying to think through sensible choices on this)